orcid-core/src/main/java/org/orcid/core/manager/impl/ProfileEntityManagerImpl.java

```diff
@@ -544,6 +550,15 @@ public Visibility getResearcherUrlDefaultVisibility(String orcid) {
     	return jpa2JaxbAdapter.getApplications(tokenDetails);
     }
     
+    @Override
+    @Cacheable(value = "orcid-hash")
```

Comment:
Why not use a session instead of a cache? I really don't like the cacheable annotation.